corehq/util/context_processors.py

```diff
@@ -57,7 +57,11 @@ def get_per_domain_context(project, request=None):
             allow_report_an_issue = True
         else:
             domain_name = request.couch_user.domain
-            allow_report_an_issue = request.couch_user.has_permission(domain_name, 'report_an_issue')
+            role = request.couch_user.get_domain_membership(domain_name).role
+            if not role and settings.SERVER_ENVIRONMENT in settings.ICDS_ENVS:
```

Comment:
Should this instead check `toggles.ICDS`?  It looks like india isn't considered an `ICDS_ENV`, so this would behave differently on India than on TCL.

Reply:
Thanks, updated!